xbmc/network/httprequesthandler/IHTTPRequestHandler.h

```diff
@@ -22,6 +22,12 @@
 #include <sys/socket.h>
 #include <sys/types.h>
 
+#if MHD_VERSION >= 0x00097002
+#define MHD_RESULT enum MHD_Result
+#else
+#define MHD_RESULT int
+#endif
+
```

Comment:
Would this work?

```
#if MHD_VERSION >= 0x00097002
using MHD_RESULT = MHD_Result;
#else
using MHD_RESULT = int;
#endif
```

If yes, I would much prefer it.

Reply:
I tested that and it worked fine. I've added a commit to this PR with that change.